mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -237,29 +237,27 @@ class CallPattern final : public SYCLToLLVMConversion<sycl::SYCLCallOp> {
                llvm::dbgs() << "\n");
 
     ModuleOp module = op.getOperation()->getParentOfType<ModuleOp>();
+    Operation *func = op->getParentOfType<LLVM::LLVMFuncOp>();
```

Comment:
This code is only needed when DEBUG info is on, it used to be inside a LLVM_DEBUG macro. I would like if you could place it at the bottom of this function. The new operation after replacement should still be able to get its parent function.